### PR TITLE
Elixir integration tests

### PR DESCRIPTION
**Pull request by danmcclain**

### CODE REVIEW

File: source/posts/2015-??-??-maintaining-session-in-phoenix-integration-tests.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+googleplus: 102648938707671188640
+github: danmcclain
+social: true
+summary: "Your server has authentication, and you want to be authenticated in your tests"
+published: true
+tags: elixir, phoenix, testing
+---
+
+I'm using [Phoenix](http://www.phoenixframework.org) to power the backends for a couple of Ember projects I am
+writing. For those applications, I've been using OAuth2 + Cookie-based sessions
```
Comment (michaeldupuisjr) [inline source/posts/2015-??-??-maintaining-session-in-phoenix-integration-tests.md]: "Cookie" is lowercased later. Lowercase here or uppercase elsewhere for consistency.

File: source/posts/2015-??-??-maintaining-session-in-phoenix-integration-tests.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+The above code copies the cookies from the first (completed) request, and
+appends them to the second request. This allows you to retrieve information
+from the session returned in the first request.
+
+
+## But wait! There is a better way
+
+With the release of Phoenix v0.13, new test helpers debuted that allow for the
+concept of connection recycling.  TL;DR: the new helpers do create a new
```
Comment (michaeldupuisjr) [inline source/posts/2015-??-??-maintaining-session-in-phoenix-integration-tests.md]: completed *connection*.

Comment (michaeldupuisjr) [inline source/posts/2015-??-??-maintaining-session-in-phoenix-integration-tests.md]: helpers create

**Comment by michaeldupuisjr** [comment]
Couple of minor comments, but otherwise, I think this looks good. :+1:

File: source/posts/2015-??-??-maintaining-session-in-phoenix-integration-tests.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+```
+
+The above code copies the cookies from the first (completed) request, and
+appends them to the second request. This allows you to retrieve information
+from the session returned in the first request.
+
+
+## But wait! There is a better way
+
+With the release of Phoenix v0.13, new test helpers debuted that allow for the
```
Comment (HeroicEric) [inline source/posts/2015-??-??-maintaining-session-in-phoenix-integration-tests.md]: I think the new test helpers were added in 0.12 

http://www.phoenixframework.org/v0.11.0/docs/upgrading-from-v0110#section-test-helpers
https://github.com/phoenixframework/phoenix/blob/v0.12.0/test/phoenix/test/conn_test.exs#L83-L114

Comment (danmcclain) [inline source/posts/2015-??-??-maintaining-session-in-phoenix-integration-tests.md]: @HeroicEric Thanks, will update!

**Pull request merged**